### ISSUE TITLE
Lack of contrast and separators in topics

### ISSUE THREAD

**Issue by denil-ct** [apple/swift-docc-render#351]
I feel like the different color given to the topics at the end of each page was a nice touch in the previous style. That along with the separators between topic headers helped to group information better. Both of those are missing in the new style. Adding the separator lines alone will help a lot in improving grouping among topics.

## New Style

<img width="680" alt="image" src="https://user-images.githubusercontent.com/95201442/172866873-73546f7f-91d4-4cfa-aa9e-aa073dee53b8.png">

## Old Style

![image](https://user-images.githubusercontent.com/95201442/172866944-e309d8a2-0617-4e38-bbd7-3d6ca8b94607.png)

**Comment by karwa** [comment]
I think this can be improved significantly by adding separators between each topic section. I've implemented it on my fork of docc-render: https://github.com/karwa/swift-docc-render/commit/0efb53301b85d65a7bcf465833fbbda029b5a334 

And here's the live documentation, if you want to see how it looks in practice:

https://karwa.github.io/swift-url/main/documentation/weburl/weburl

![image](https://user-images.githubusercontent.com/5254025/179368563-a5f6b2f9-7dc8-434f-a8a0-df6a225866d3.png)

**Comment by denil-ct** [comment]
Yes @karwa , this looks great and much more readable at a glance.
